locale/en/docs/index.md

```diff
@@ -15,9 +15,9 @@ There are several types of documentation available on this website:
 
 ## API reference documentation
 
-The [API reference documentation](https://nodejs.org/api/) provides detailed information about a function or object in Node.js. This documentation indicates what arguments a method accepts, the return value of that method, and what errors may be related to that method. It also indicates which methods are available for different versions of Node.js.
+The [API reference documentation](https://nodejs.org/api/) provides detailed information about a function or object in Node.js such as: what arguments a method accepts, the return value of that method, and what errors may be related to that method. It also describes the difference in available methods provided by each version  of Node.js
```

Comment:
The period was fine as it was.

Reply:
periodS* were*

Reply:
The point @XhmikosR was trying to make was that the change here removes the period at the end of the last sentence. In other words, it introduces an error.